client/webpack.config.js

```diff
@@ -119,6 +125,23 @@ module.exports = {
 
 // helpers //////////////////////
 
+function sentryIntegration() {
+  if (!SENTRY_DSN || NODE_ENV !== 'production') {
```

Comment:
Any reason why we forcefully exclude NODE_ENV=development here?

I'd assume that the thing is activated, no matter what if we provide the SENTRY_DSN env var.

That would allow us to easily test the app:

```
SENTRY_DSN=FOO npm run dev
```

Reply:
So we can still have error tracking via flags or via manually entering an ENV variable but we don't want to pollute source maps inside Sentry. That's why I disabled source map uploading via Webpack for development environments.

Reply:
Just because we don't upload source maps does not mean that we won't be able to send errors to Sentry for dev though. Sentry still works if we manually set a DSN in development.

Reply:
Got it. 

I'd personally assume that source maps worked if I test this thing in DEV, too. Similar to previous projects we worked on => least differences between prod and dev.

To get this done I'd tag the source maps in `dev` with `dev` as a release or `dev-last-commit-hash`. Would that be an option?

Reply:
Done via this commit: 05903d633dabf9aa5b442507c71c2a5da511a1d1

We tag the dev releases as "dev". Source map uploading can be tested in development given that `SENTRY_DSN`, `SENTRY_AUTH_TOKEN`, `SENTRY_ORG` and `SENTRY_PROJECT` environment variables are manually injected.